### PR TITLE
Update hardcoded headers (foundation, classic, kovan,  xdai, ewc, ...)

### PR DESCRIPTION
**Pull request by joshua-mir**
still syncing: poacore, poasokol, volta

### CODE REVIEW

File: ethcore/res/ethereum/ewc.json [hunk 1]
```diff
@@ -1,213 +1,954 @@
...
...structor": "0x60806040523480156200001157600080fd5b5060405162002d7138038062002d71833981018060405260608110156200003757600080fd5b81019080805190602001909291908051906020019092919080516401000000008111156200006457600080fd5b828101905060208101848111156200007b57600080fd5b81518560208202830111640100000000821117156200009957600080fd5b5050929190505050336000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff1602179055506000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16600073ffffffffffffffffffffffffffffffffffffffff167f8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e060405160405180910390a3600073ffffffffffffffffffffffffffffffffffffffff168273ffffffffffffffffffffffffffffffffffffffff161415620001e5576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252602481526020018062002d216024913960400191505060405180910390fd5b60018151101562000242576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252602c81526020018062002d45602c913960400191505060405180910390fd5b62000253836200046160201b6020
```
Comment (niklasad1) [inline ethcore/res/ethereum/ewc.json]: double indentation?

Comment (niklasad1) [inline ethcore/res/ethereum/ewc.json]: indentation

File: ethcore/res/ethereum/kovan.json [hunk 2]
```diff
@@ -77,9 +77,9 @@
...
-		"totalDifficulty": "3571337622391237938633151520660827524104528124",
+	"hardcodedSync":{
```
Comment (niklasad1) [inline ethcore/res/ethereum/kovan.json]:
```suggestion
	"hardcodedSync": {
```

File: ethcore/res/ethereum/poacore.json [hunk 3]
```diff
@@ -49,6 +49,5354 @@
 		"difficulty": "0x20000",
 		"gasLimit": "0x663BE0"
 	},
+	"hardcodedSync":{
```
Comment (niklasad1) [inline ethcore/res/ethereum/poacore.json]:
```suggestion
	"hardcodedSync": {
```

File: ethcore/res/ethereum/volta.json [hunk 4]
```diff
@@ -1,207 +1,1306 @@
...
...ffffffffffffff16815260200190815260200160002060000160006101000a81548160ff02191690836003811115620003b257fe5b02179055508060036000848481518110620003c957fe5b602002602001015173ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff1681526020019081526020016000206001018190555080806001019150506200027c565b508060029080519060200190620004399291906200065c565b50600260019080546200044e929190620006eb565b506001600060146101000a81548160ff02191690831515021790555050505062000788565b600073ffffffffffffffffffffffffffffffffffffffff168173ffffffffffffffffffffffffffffffffffffffff16141562000517576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252601f8152602001807f4e6577206f776e657220616464726573732063616e6e6f74206265203078300081525060200191505060405180910390fd5b8073ffffffffffffffffffffffffffffffffffffffff166000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff167f8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e060405160405180910390a3806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555050565b
```
Comment (niklasad1) [inline ethcore/res/ethereum/volta.json]: double indentation? I guess it was spaces before

**Comment by dvdplm** [comment]
> still syncing: poacore, poasokol, volta

These are done right?

**Comment by joshua-mir** [comment]
Yep👍

**Pull request merged**